From 7d8d738d87212e915b51e8b85626355f4470c42a Mon Sep 17 00:00:00 2001
From: haiqiang yao <haiqiangx.yao@intel.com>
Date: Tue, 11 Aug 2015 16:13:04 +0800
Subject: [PATCH 461/462] ov2722:Change 480p sensor setting from 736x480 to
 1632x1092

The quality of offline captured 480p image (with sensor 736x496
setting) is very bad.

This patch improve the quality by changing 480p preview sensor
setting to 1632x1092.

Change-Id: I3df73599e033a8f6bad3dd14d8614ec3bfc6572d
Signed-off-by: haiqiang yao <haiqiangx.yao@intel.com>
Tracked-On: https://jira01.devtools.intel.com/browse/GMINL-15075
Reviewed-on: https://icggerrit.ir.intel.com/68784
Approver: Bingbu Cao <bingbu.cao@intel.com>
Reviewed-by: Bingbu Cao <bingbu.cao@intel.com>
Validation-Android: Wei Tao <wei.tao@intel.com>
Build: IMIN (SC)
Integrator: WeiX Meng <weix.meng@intel.com>
Build: WeiX Meng <weix.meng@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
Maintainer: WeiX Meng <weix.meng@intel.com>
---
 .../external_drivers/camera/drivers/media/i2c/ov2722.h   | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov2722.h b/drivers/external_drivers/camera/drivers/media/i2c/ov2722.h
index 4cf00f7..7756aaa5 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov2722.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov2722.h
@@ -1118,21 +1118,21 @@ static struct ov2722_reg const ov2722_720p_30fps[] = {
 
 struct ov2722_resolution ov2722_res_preview[] = {
 	{
-		.desc = "ov2722_480P_30fps",
-		.width = 736,
-		.height = 496,
+		.desc = "ov2722_1632_1092_30fps",
+		.width = 1632,
+		.height = 1092,
 		.fps = 30,
-		.pix_clk_freq = 73,
+		.pix_clk_freq = 85,
 		.used = 0,
-		.pixels_per_line = 2048,
-		.lines_per_frame = 1184,
+		.pixels_per_line = 2260,
+		.lines_per_frame = 1244,
 		.bin_factor_x = 1,
 		.bin_factor_y = 1,
 		.bin_mode = 0,
 		.skip_frames = 3,
-		.regs = ov2722_480P_30fps,
+		.regs = ov2722_1632_1092_30fps,
+		.mipi_freq = 422400,
 	},
-
 	{
 		.desc = "ov2722_1452_1092_30fps",
 		.width = 1452,
-- 
1.9.1

